### ISSUE TITLE
Can we use other models to estimate the social interactions since it is not necessarily linear? 

### ISSUE THREAD

**Issue by ruixili** [uchicago-computation-workshop/steven_durlauf#26]

**Comment by sdurlauf** [comment]
Absolutely!  The identification conditions are different. The linear model poses greatest challenges for identification and is the most common one in empirical work, hence the focus in the JPE paper.

**Comment by gretaheng** [comment]
I used clustering methods could be used in the initial stage, followed by linear models for each cluster. That works great on my weibo data.

**Comment by sdurlauf** [comment]
Interesting! Would be interested in seeing the paper. In my older work on economics growth, I argued that countries with similar levels of development obeyed distinct linear growth models. Here groupings emerged from tree methods.
